app/models/organisation.rb

```diff
@@ -112,6 +112,12 @@ class Organisation < ApplicationRecord
 
   has_many :offsite_links, as: :parent
 
+  def featured_documents_display_limit
```

Comment:
I'd have thought this method would need to be updated: [featured_documents_display_limit](https://github.com/alphagov/whitehall/blob/06a1c9e2bbbc31a0b34807f8b8b8a167b3272c53/app/models/organisation.rb#L510) ?

Reply:
Lol, that's what Ashley thought as well. Featured links and featured documents are two separate things.

Reply:
Oh! That's my fault, Ash got that from my investigations. Oops!